packages/ckeditor5-dev-env/lib/release-tools/tasks/bump-year.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+/**
+ * Updates year in all licenses in the provided directory, based on provided glob patterns.
+ *
+ * Replaces:
+ * 	Copyright (c) [initial year]-xxxx
+ *
+ * With:
+ *	Copyright (c) [initial year]-[current year]
+ *
+ * @param {Object} params
+ * @param {String} params.cwd Current working directory from which all paths will be resolved.
+ * @param {Number} [params.initialYear='2003'] Year from which the licenses should begin.
+ * @param {Array} params.globPatterns array of objects, where each object has string property 'pattern',
+ * and optionally 'options' property.
+ */
+module.exports = function bumpYear( params ) {
+	if ( !params.initialYear ) {
+		params.initialYear = '2003';
+	}
+
+	console.log( chalk.green( 'Looking for files to update...' ) );
+
+	const filesToUpdate = params.globPatterns
+		.map( globPattern => {
+			const options = globPattern.options || {};
+
+			const defaultIgnore = [
+				'**/node_modules/**'
+			];
+
+			options.nodir = options.nodir || true;
```

Comment:
What is the point of `options.nodir = options.nodir || true`?

The `options.nodir` will be always `true` in this ☝🏻 instruction, even if received `globPattern.options.nodir` is `false`.

Reply:
Right, my bad. I wanted to have default `true` value, but still prioritise whatever value would be passed from the options.

I've replaced that with `options.nodir = options.nodir == null ? options.nodir : true`, this will only use default value for `undefined` and `null`, but not for `false` as `false != null` returns `true`,

Reply:
But still, why do you want to allow directories to be targeted, since a few lines below you expect that all the returned results from `glob` are files? You are trying to open and read the content from all the found paths, so when the given path points to a directory, the script will crash.

I think that the `nodir` should have a hardcoded value equal `true`, because the script is able to work only on files.